### PR TITLE
Convert nuspecs to sdk projects

### PR DESCRIPTION
**Pull request by ViktorHofer**
Checked all packages. Two packages don't follow the standard content include pattern which I prefer to change on the consuming side in corefx to be consistent.

### CODE REVIEW

**Comment by ViktorHofer** [comment]
@ericstj I'm okay with targeting netstandard2.0 in the test projects now with the switch to SDK projects. any concerns?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# CoreFx-TestData
+
+This repository contains test assets which are too large to be checked into corefx directly. Packages are produced and uploaded to the dotnet blob feed (https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
```
Comment (joshfree) [inline README.md]: nit: test assets that are binary file or too large...

**Comment by ericstj** [comment]
> Two packages don't follow the standard content include pattern which I prefer to change on the consuming side in corefx to be consistent.

For those two packages, make sure you bump the version.

> I'm okay with targeting netstandard2.0 in the test projects now with the switch to SDK projects. any concerns?

It shouldn't matter if we're only packing content.  Let me know if you're noticing anything about this showing up in the packages.

File: README.md [hunk 2]
```diff
@@ -1,151 +1,11 @@
...
-Locate the ````<version>```` element. If the current version is:
-
-````xml
-<version>1.0.7-prerelease</version>
-````
-
-then bump it to:
-
-````xml
-<version>1.0.8-prerelease</version>
```
Comment (ericstj) [inline README.md]: What's the versioning story now?

**Comment by ViktorHofer** [comment]
> It shouldn't matter if we're only packing content. Let me know if you're noticing anything about this showing up in the packages.

Right, it shouldn't matter. But still the target framework in the nupkg metadata will show netstandard2.0.

File: Directory.Build.props [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IsPackable>true</IsPackable>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <Title>Test data for $(MSBuildProjectName.Replace('.TestData', '')) tests</Title>
+    <Description>This package provides test data that is used during the execution of the $(MSBuildProjectName.Replace('.TestData', '')) tests.</Description>
+    <Authors>Microsoft</Authors>
+    <Copyright>Copyright © Microsoft Corporation</Copyright>
+    <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</PackageLicenseUrl>
```
Comment (safern) [inline Directory.Build.props]: I think this property is being deprecated, but don't remember 100%.

File: README.md [hunk 4]
```diff
@@ -1,151 +1,11 @@
...
 
-1. Clone the corefx-testdata repo.
-
-2. In the root folder of the corefx-testdata repo, locate the folder with the namespace of the tests you are working on and place your input files inside this folder in the appropriate location.
-
-### **Example**
-
-Let's say you are adding tests in the ````System.IO.Compression```` namespace for the GZip features. You can save your new file called example.gz inside the folder:
-
-````corefx-testdata/System.IO.Compression.TestData/GZipTestData/````
```
Comment (safern) [inline README.md]: Why are we deliting the consumption in corefx? We're planning on not bumping versions?

File: Directory.Build.props (hunk 3 above)
Comment (ViktorHofer) [reply Directory.Build.props]: Right it is, but I'm fine with using it at the moment. I quickly tried to find out what to use instead but it wasn't 100% obvious.

File: System.IO.Compression.TestData/System.IO.Compression.TestData.csproj [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageVersion>1.0.7-prerelease</PackageVersion>
```
Comment (safern) [inline System.IO.Compression.TestData/System.IO.Compression.TestData.csproj]: Why some of them are prerelease?

Comment (ViktorHofer) [reply System.IO.Compression.TestData/System.IO.Compression.TestData.csproj]: Don't ask me 😂 this whole repo was a mess. I will bump all the versions with this PR.

File: README.md (hunk 2 above)
Comment (ViktorHofer) [reply README.md]: I just added documentation for that and bumped all package versions to get into a clean state with prerelease labels.

File: README.md (hunk 4 above)
Comment (ViktorHofer) [reply README.md]: Added some info in the docs.

File: System.IO.Compression.TestData/System.IO.Compression.TestData.csproj (hunk 5 above)
Comment (ViktorHofer) [reply System.IO.Compression.TestData/System.IO.Compression.TestData.csproj]: Fixed.

**Comment by carlossanlop** [comment]
These are the instructions in the readme from this PR:

> 1. Make changes in the respective project and increments its version (usually just the patch fragment)
> 2. Run dotnet pack on the project and find the produced nuget package in the project's bin folder.
> 3. Use this build definition to upload the package to the dotnet blob feed: https://github.com/dotnet/core-eng/tree/master/Documentation/Tools/dotnet-core-push-oneoff-package. Make sure to clear the myget feed in the queue time variable section.
> 4. Consume the updated package in corefx by incrementing the PackageReference version attribute: https://github.com/dotnet/corefx/blob/8d79b6117e9d584eebb8b6933bba83dd514010ca/external/test-runtime/XUnit.Runtime.depproj#L47-L54

Let's say I was a new contributor. I would have trouble understanding these things:

1. "Make changes in the respective project". What kinds of changes should I make? What does "project" mean here?

2. "Run dotnet pack on the project". Do we still need to use "dotnet" and not "nuget.exe"? I found this problem and described it in my readme. I also gave a link to download nuget client tools.

3. We need an example of this step. Even though there is a link to the core-eng instructions, there are some steps there that are for corpnet people only, so external people would not know what do to here. Also, how can the myget feed be cleared and why do we need to do that?

4. I don't think linking to a file with highlighted lines is a good idea. The file could be changed, and the user won't know what to do. Instead, we should have an example.

These instructions also don't tell the user how to consume a corefx-testdata locally, *before* sending their PRs. I had those instructions too.

Here's my suggestion: let's bring back the readme file I wrote, but we modify the steps that are different now, making sure each step has a description of why it has to be done and has an example.

**Comment by ViktorHofer** [comment]
Hey @carlossanlop, sorry I currently don't have cycles for this. Would you be willing to send another PR with your suggestions?

**Comment by carlossanlop** [comment]
I could merge the readme file I originally uploaded, but I am unfamliar with the new instructions, so it would be better if you do those changes.

**Comment by ViktorHofer** [comment]
Ok I'll update the steps but probably not in the week or so.

**Pull request merged**